src/entities/expenses/api/amount.api.ts

```diff
@@ -0,0 +1,9 @@
+import { type AxiosResponse } from 'axios';
+import { axiosInstance } from '@shared/api/axios';
+
+export const amountApi = async (userId: string): Promise<AxiosResponse> => {
+  const response = await axiosInstance.get<AxiosResponse>(
```

Comment:
We need to know what response we had from API. Please add interface or type here